### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by franklinben23**
In this part of the testing project, we made the test cases for the functions responsible for the status and content updates.

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Because the file wasn't changed I'm unable to leave an inline comment, however in the 'index.js' file in the '__mocks__' folder there is a lot of commented out code, please remove this before requesting for another code review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by franklinben23** [comment]
Is there anything else that needs to be changed? is there something i need to do to allow you to make inline comments if needed?

File: __mocks__/index.js [hunk 1]
```diff
@@ -39,49 +39,7 @@ const render = () => {
...
-    //   li.appendChild(saveBtn);
-    //   saveBtn.addEventListener('click', () => {
-    //     task.name = newInput.value;
-    //     record();
-    //     render();
-    //   });
-    // });
   });
 };
 /* eslint-disable */
```
Comment (D4nielJ) [inline __mocks__/index.js:45]: [OPTIONAL] Please try to disable the least amount of `eslint` rules, instead you can always search for how to fix these. Remember that these rules are here to help us keep the code clean, free of bugs, and standardized across teammates.

Hint: You can fix this one using: `export default component;`. [Read more about ES6 exports here.](https://devhints.io/es6#modules)

**Review by D4nielJ — APPROVED** [review]
## Hello @franklinben23 and @angelacuahutle, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**